warehouse/query-core/src/main/java/datawave/query/predicate/EventDataQueryFieldFilter.java

```diff
@@ -9,26 +9,34 @@
 import org.apache.commons.jexl2.parser.ASTIdentifier;
 import org.apache.commons.jexl2.parser.ASTJexlScript;
 
+import javax.annotation.Nullable;
 import java.util.Map;
 import java.util.Set;
+import java.util.SortedSet;
 
 /**
  * This filter will filter event data keys by only those fields that are required in the specified query.
  */
-public class EventDataQueryFieldFilter extends KeyProjection implements EventDataQueryFilter {
+public class EventDataQueryFieldFilter implements EventDataQueryFilter {
     private Set<String> nonEventFields;
 
-    public EventDataQueryFieldFilter() {
-        super();
-        // empty white list and black list
-    }
+    private final KeyProjection keyProjection;
 
     public EventDataQueryFieldFilter(EventDataQueryFieldFilter other) {
-        super(other);
         this.nonEventFields = other.nonEventFields;
```

Comment:
I think this still needs a call to `super(other)`

Reply:
adding back.

Reply:
I take that back, good change.